### PR TITLE
Use app container for token provider

### PR DESCRIPTION
**Pull request by StephenThomson**
# What

Previously a custom container was used to pass in token provider, now we use the app container.

# Why

Consistency
To support the work to ensure login is not possible with tokens from archived wards.

# Screenshots

# Notes

### CODE REVIEW

File: pageTests/wards/visit-start.test.js [hunk 1]
```diff
@@ -16,7 +16,17 @@ describe("visit-start", () => {
...
+      const authenticationToken = {
+        foo: true,
+      };
+      const tokenProvider = {
+        validate: jest.fn(() => authenticationToken),
+      };
+      const container = {
+        getTokenProvider: () => tokenProvider,
+      };
+      console.log(container);
```
Comment (lukemorton) [inline pageTests/wards/visit-start.test.js:28]: I spy the debug

Comment (StephenThomson) [reply pageTests/wards/visit-start.test.js:28]: 😞

**Comment by StephenThomson** [comment]
I think there might be a problem, need to have a look with someone with heroku access tomorrow

**Comment by StephenThomson** [comment]
The problem was just that the Notify API key used in staging will only text team members and I wasn't a team member.

**Pull request merged**